tron/utils/rookout.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+import platform
+from typing import Optional
+
+import rook
+import staticconf  # type: ignore
+
+import tron
+from tron.config.static_config import get_config_watcher
+from tron.config.static_config import NAMESPACE
```

Comment:
the public tests/itests are probably going to be broken 'cause srv-configs aren't available on GHA

we could figure out a good way to mock these in both - but maybe we should move the rookout config out of here and either into tron's soaconfigs or just environment variables that we set in puppet in the systemd unit file. 

srv-configs really shines for things that can change without a "reboot", but this rookout code will only ever be called at startup so we don't necessarily need to deal with getting staticconf to work with our public tests if we move this config out of there :p

Reply:
I've modified the tests to mock the functions

Reply:
Okay, I understand what you mean. What do you think is the better option?
Option 1:
1. Add the following `ROOKOUT_TOKEN_PATH, ROOKOUT_HOST, ROOKOUT_PORT, ROOKOUT_ENABLE` in tron's soaconfigs.
2. Modifying tron's api to trigger rookout reload whenever any of these variables change.

Option2:
1. Add the following `ROOKOUT_TOKEN_PATH, ROOKOUT_HOST, ROOKOUT_PORT, ROOKOUT_ENABLE`  as env variables in puppet so when these get modified it'll just restart tron with the modified variables.

Option 2 would definitely be the easier approach. and I don't think these variables would be changing often at all.

Reply:
option 2 sounds best imo - especially since the rookout docs make it seem like the only thing that rookout supports reloading for is updated labels